Donation-platform/src/main/java/com/google/FindTrendingCharities.java

```diff
@@ -94,37 +103,49 @@ private double calcCharityTrendingScore(Charity charity) {
             avgReview = userRating;
         }
         //TODO: Integrate db with correct method to retreive tags for one charity
-        Collection<String> tags = charity.getTags();
-        double charityTagsScore = getTagTrendingScore(tags);
+        Collection<Tag> tags = new ArrayList<>();
+        try {
+           tags = db.getTagObjectsByIds(charity.getCategories());
+        }
+        catch (Exception e) {
```

Comment:
Be more specific in the Exception. This will hide all errors and even unrelated ones.